### DISCUSSION TITLE
Can't convert mkv to mp4 (remultiplexing). Why?

### DISCUSSION THREAD

**Discussion by mfifty23** [obsproject/obs-studio#6044] [General]
It says: "Alright, done, but the file may be not full", so there is the file, which is 0 kb. What is it and how to fix it?

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
